app/models/legal_aid_application.rb

```diff
@@ -155,6 +155,10 @@ def submission_date
     (transaction_period_finish_at || Time.now.utc).to_date
   end
 
+  def passported?
+    benefit_check_result&.positive?
+  end
```

Comment:
There's already `applicant_receives_benefit?` on line 104 so I suggest using that. Feel free to replace `.` by `&.`. My [PR 771](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/771/files#diff-f0eba5ecee5c3702fb86e094c801215eR112) also does it.

In my opinion, we don't need the term `passported?` but we can always use `applicant_receives_benefit?` instead.
It's clearer.

However, if you really want to use the term `passported?` you can add it as an alias of `applicant_receives_benefit?` :
```ruby
  def applicant_receives_benefit?
    benefit_check_result&.positive?
  end
  alias_method :passported?, :applicant_receives_benefit?
```

Reply:
`applicant_receives_benefit?` is slightly misleading, in that an applicant may receive a benefit, but it is not a qualifying benefit, so he/she is not passported.  I would prefer `applicant_receives_qualifying_benefit?`, or using the term `passported?` which is jargon, but understood by everyone.

In AP-885, I have put a `#passported?` method on `LegalAidApplication` - so we probably need to standardise on one method and use it consistently.